diagnostic.md

```diff
@@ -13,7 +19,7 @@ Using the diagram in the `README` file, what are two ways you could return ALL
 list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
-<!-- your answer starts here -->
+console.log( $( "li" )[ 0 ] );
```

Comment:
The call the `console.log` is unnecessary and `$('li')[0]` will only return the first `<li>` element.  From `<ol>`, two possible ways to select `<li>` elements are `$('ol > li');` and `$('ol').children();`.  Some other possible ways are `$('li');`, `$('body').find('li');`, and `$('body').children('ol').children('li');`.